docs-source/content/userguide/managing-domains/model-in-image/runtime-updates.md

```diff
@@ -0,0 +1,325 @@
+
+_Can we use custom configuration overrides to do the updates instead?_
+
+No. Custom configuration overrides, which are WebLogic configuration overrides specified using a domain resource `configuration.overridesConfigMap`, as described in [Configuration overrides]({{< relref "/userguide/managing-domains/configoverrides/_index.md" >}}), aren't supported in combination with Model in Image. Model in Image will generate an error if custom overrides are specified. This should not be a concern because model file, secret, and/or model image updates are simpler and more flexible than custom configuration override updates. Unlike configuration overrides, the syntax for a model file update exactly matches the syntax for specifying your model file in the first place.
+
+
+#### Supported and unsupported updates
+
+ - You can add new MBeans or resources simply by specifying their corresponding model file YAML snippet along with their parent bean hierarchy. See [Example of adding a data source](#example-of-adding-a-datasource).
+
+ - You can recreate, change, or add secrets that your model depends on. For example, you can change a database password secret.
+
+ - You can change or add environment variables that your model macros may depend on (macros that use the `@@ENV:myenvvar@@` syntax).
+
+ - You can remove a named MBean or resource by specifying a model file with a `!` symbol just before the bean or resource name. For example, if you have a data source named `mynewdatasource` defined in your model, it can be removed by specifying a small model file that loads after the model file that defines the data source, where the small model file looks like this:
+
+  ```
+  resources:
+    JDBCSystemResource:
+      !mynewdatasource:
+  ```
+
+  For more information, see [Declaring Named MBeans to Delete](https://github.com/oracle/weblogic-deploy-tooling#declaring-named-mbeans-to-delete) in the WebLogic Deploying Tooling documentation.
+
+ - You can add or alter an MBean attribute by specifying a YAML snippet along with its parent bean hierarchy that references an existing MBean and the attribute. For example, to add or alter the max capacity of a data source named `mynewdatasource`:
+
+  ```
+  resources:
+    JDBCSystemResource:
+      mynewdatasource:
+        JdbcResource:
+          JDBCConnectionPoolParams:
+            MaxCapacity: 5
+  ```
+
+  For more information, see [Using Multiple Models](https://github.com/oracle/weblogic-deploy-tooling#using-multiple-models) in the WebLogic Deploy Tooling documentation.
+
+ - There is no way to directly delete an attribute from an MBean that's already been specified by a model file. The work-around is to do this using two model files: (a) add a model file that deletes the named bean/resource that is a parent to the attribute you want to delete, and (b) add another  subsequent model file that fully defines the named bean/resource but without the attribute you want to delete.
```

Comment:
extra space "another  subsequent"

Reply:
The extra space that you see here, does not show up in the UI, "(b) add another subsequent model file that..."